packages/block-editor/src/components/inserter/menu.js

```diff
@@ -167,7 +167,8 @@ function InserterMenu( {
 							setFilterValue( value );
 						} }
 						value={ filterValue }
-						placeholder={ __( 'Search' ) }
+						label={ __( 'Search' ) }
+						placeholder={ __( 'Search for blocks and patterns' ) }
```

Comment:
we can also search for reusable blocks, not sure if it counts as "blocks". 

The placeholder change seems like a design decision, so let's try to get some opinions.

Reply:
Thanks for the ping. You can indeed search for blocks, patterns, and reusable blocks.

However for brevity, I would think it okay to lump in reusable blocks with blocks, for a few reasons:

- Reusable blocks only show up when you first create one.
- When that happens, there's also a separate tab for it. 

So in the name of brevity, this seems okay to try 👍 👍

Reply:
What I don't like personally is that sometimes there's no patterns... and "Search" is more generic.  I don't want us to introduce conditions (so performance impact calling selectors) just to tweak the placeholder accurately. Isn't there another solution while still keeping the "Search" placeholder?

Reply:
Phrase-wise, plain old "Search" would be fine for me also. I'll defer if there's a technical underpinning to the change.

Reply:
@alexstine suggested "Search for blocks" on https://github.com/WordPress/gutenberg/pull/26938#issuecomment-756199150. Since patterns are also collections of blocks and the button label is "Add block", it sounds generic enough, but still more descriptive than just "Search".